packages/color-block/src/components/drops/DropsItem.spec.ct.tsx

```diff
@@ -0,0 +1,102 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import { mount } from '@cypress/react';
+import { FrontifyColorDummy } from '@frontify/app-bridge';
+import { DropsItem } from './DropsItem';
```

Comment:
Space again. Please review all files and fix the imports so I do not repeat myself.